### ISSUE TITLE
Include/Exclude in the response?

### ISSUE THREAD

**Issue by jasonbahl** [apollographql/apollo-tracing#5]
Has there been any discussion on how the consumer can specify whether to include tracing in the response? 

My initial thought for my implementation would be to check the request headers to see if tracing is being requested or not.

**Comment by jasonbahl** [comment]
Would also be good to create a standard so tools like GraphiQL can integrate and send a header (or something to that tune) that asks to `includeTracing`. 

The standard could be a header, or an addition to the request payload. . .like: 

```
{
  operationName: 'queryWithTracing',
  query: 'query queryWithTracing { posts { edges { node { id } } } }',
  variables: {},
  extensions: {
     tracing: true
  }
}
```

**Comment by martijnwalraven** [comment]
Yeah, I think it makes sense to add support for something like this. We may not want to depend on HTTP because people might be using alternative transports (such as WebSockets), but an `extensions` field as part of the request payload makes sense to me.

**Comment by Kosta-Github** [comment]
I am using something like this in my queries:
{
    RuntimeStatistics {
        IncludeURLs
        RequestTimings
    }
}
The corresponding resolvers having the side effect to inject the requested statistics into the extension prop at the end of the query...
